src/bin/bench.rs

```diff
@@ -72,6 +73,7 @@ Options:
     --no-fail-fast               Run all benchmarks regardless of failure
     --frozen                     Require Cargo.lock and cache are up to date
     --locked                     Require Cargo.lock is up to date
+    -Z FLAG ...                  Unstable (nightly-only) flags to Cargo
```

Comment:
It's possible that it's better to hide this from help, just to avoid having the `unstable` word in the help for the main Rust tool, but looks like this can be difficult to achieve with docopt, so we can just skip it altogether.

Reply:
Yeah that's how I felt, and another use case for not using docopt eventually!